client/state/current-user/selectors.js

```diff
@@ -32,6 +32,15 @@ export function getCurrentUserLocale( state ) {
 
 	return user.localeSlug || null;
 }
+/**
+ * Returns the currency code for the current user.
+ *
+ * @param  {Object}  state  Global state tree
+ * @return {?String}        Current currency code
+ */
+export function getCurrentUserCurrencyCode( state ) {
+	return get( state.currentUser, 'currencyCode', null );
```

Comment:
I don't think you'd need either the `get` call here nor the `null` fallback? Should just be `return state.currentUser.currencyCode`, no?